tests/testMinimize.py

```diff
@@ -72,22 +72,19 @@ def testMinimize2(self):
 
         print("modelParams=", modelParams)
         print("fitParams  =", fitResults.parameterList)
-        self.assert_(fitResults.isValid, "fit failed")
+        self.assertTrue(fitResults.isValid, "fit failed")
         if not numpy.allclose(modelParams, fitResults.parameterList):
```

Comment:
I suggest using assertFloatsNotEqual (from lsst.utils.tests.TestCase) or the numpy.testing equivalent